src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/EthAddressInput/index.tsx

```diff
@@ -32,7 +32,7 @@ const EthAddressInput = ({
   name,
   onScannedValue,
   text,
-}: EthAddressInputProps) => {
+}: EthAddressInputProps): JSX.Element => {
```

Comment:
We should go with `React.ReactElement`

https://github.com/gnosis/safe-react/pull/1074#discussion_r455333147

You can find an explanation in the commit [16711361](https://github.com/gnosis/safe-react/pull/1074/commits/1671136115b3267a5ac3371b638c8d8ea975bf4a) message